### PR TITLE
PUD-765 - Add NSY_REMOVE_WARRANT_EMAIL document category

### PR DESCRIPTION
**Pull request by jonnywyatt**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/DocumentCategory.kt [hunk 1]
```diff
@@ -18,6 +18,7 @@ enum class DocumentCategory(val uploaded: Boolean, val versioned: Boolean) {
   RECALL_NOTIFICATION(false, true),
   RECALL_NOTIFICATION_EMAIL(true, true),
   RECALL_REQUEST_EMAIL(true, true),
+  NSY_REMOVE_WARRANT_EMAIL(true, true),
```
Comment (philsegal) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/DocumentCategory.kt:21]: They're alphabetised at the moment...

**Review by philsegal — CHANGES_REQUESTED** [review]
Please add in alphabetical order :)

File: src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/DocumentCategory.kt (hunk 1 above)
Comment (jonnywyatt) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/DocumentCategory.kt]: done 😀

Comment (jonnywyatt) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/db/DocumentCategory.kt]: done 😀

**Review by philsegal — APPROVED** [review]

**Pull request merged**